source/_components/alarm_control_panel.paradox.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Paradox Alarm Control Panel"
+description: "Instructions how to integrate Paradox alarms into Home Assistant."
+date: 2016-10-07 12:17
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: paradox.png
+ha_category: Alarm
+ha_release: 0.30
+ha_iot_class: "Local Push"
+---
+
+The `paradox` alarm control platform is a sub module of the [Paradox hub](/components/paradox/) which provides integration with Paradox alarms utilizing the PRT3 module.
+
+Setting up the hub allows you to enable this platform, which exposes all of your areas/partitions as alarm control panels. Doing so provides visibility of the areas through the UI as well as the ability to trigger automation actions when something happens like an area being armed/disarmed.
```

Comment:
UI on your control panels or in the HA interface?  I would recommend mentioning either the HA interface or specifying which interface.